package-lock.json

```diff
@@ -128550,10 +128550,402 @@
         "eslint": "8.56.0",
         "eslint-config-custom-server": "*",
         "tsconfig": "*",
+        "tsx": "^4.19.2",
```

Comment:
Noticed that this version is still unpinned. May need to reset package-lock changes and run npm i again with the pinned version!

Reply:
ah okay will do!